src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -412,7 +412,9 @@ private boolean shouldContinueTabulating() {
     int numEliminatedCandidates = candidateToRoundEliminated.size();
     int numWinnersDeclared = winnerToRound.size();
     // apply config setting if specified
```

Comment:
let's just delete this comment; it's ambiguous and doesn't add anything anyway

Reply:
Can do. (Note: It wasn't part of this PR.)